diff --git a/CMakeLists.txt b/CMakeLists.txt
index 460d13a..079d9d9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -241,7 +241,7 @@ set(ROCKSDB_LIBS rocksdblib${ARTIFACT_SUFFIX})
 set(LIBS ${ROCKSDB_LIBS} ${THIRDPARTY_LIBS} ${SYSTEM_LIBS})
 
 add_subdirectory(third-party/gtest-1.7.0/fused-src/gtest)
-add_subdirectory(tools)
+# add_subdirectory(tools)
 
 # Main library source code
 
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 66717d5..9112436 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -726,7 +726,7 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+#add_subdirectory(tools)
 
 # Installation and packaging for Linux
 if (NOT WIN32)
