diff --git a/thirdparty.inc b/thirdparty.inc
index f40b81fec..31f7a69f8 100644
--- a/thirdparty.inc
+++ b/thirdparty.inc
@@ -51,10 +51,12 @@ endif ()
 #
 # Edit these 4 lines to define paths to Snappy
 #
-set(SNAPPY_HOME $ENV{THIRDPARTY_HOME}/Snappy.Library)
-set(SNAPPY_INCLUDE ${SNAPPY_HOME}/build/native/inc/inc)
-set(SNAPPY_LIB_DEBUG ${SNAPPY_HOME}/lib/native/debug/amd64/snappy.lib)
-set(SNAPPY_LIB_RELEASE ${SNAPPY_HOME}/lib/native/retail/amd64/snappy.lib)
+get_filename_component(SNAPPY_HOME "${CMAKE_CURRENT_LIST_DIR}/../snappy" ABSOLUTE)
+set(SNAPPY_INCLUDE ${SNAPPY_HOME} ${SNAPPY_HOME}/build)
+set(SNAPPY_LIB_DEBUG ${SNAPPY_HOME}/build/Debug/snappy.lib)
+set(SNAPPY_LIB_RELEASE ${SNAPPY_HOME}/build/Release/snappy.lib)
+
+
 
 # For compatibility
 if(SNAPPY)
